addons/binding/org.openhab.binding.enigma2/src/main/java/org/openhab/binding/enigma2/internal/Enigma2RemoteKey.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.enigma2.internal;
+
+/**
+ * The {@link Enigma2RemoteKey} represents the remote keys of an enigma2 device
+ *
+ * @author Thomas Traunbauer - Initial contribution
+ */
+public enum Enigma2RemoteKey {
+    // Source found on
+    // https://dream.reichholf.net/wiki/Webinterface_Befehle#Tasten_der_Fernsteuerung:_.2Fcgi-bin.2Frc.3F.3Ckey.3E:.3Cduration.3E:.3Creptime.3E.2C.3Ckey....3E
```

Comment:
Der Inhalt ist verfügbar unter der Lizenz GNU Free Documentation License 1.2, sofern nicht anders angegeben.

How should i note that?